### PR TITLE
feat/auth google: sign in with Google to get  name and email user

### PR DESCRIPTION
**Pull request by doohanas**
## Task
1. https://airtable.com/appdN4AdnU8FNs94M/tblRtWbyq60Fx1m6v/viwlpGjUkaHCytKMk/recstd7hHQQHIfEWP?blocks=hide

## Changes

1. Sign in with Google to get  name and email user

## Preview

1. Sign in with Google to get  name and email user
preview

## Evidence
title: feat/auth google: sign in with Google to get  name and email user
project: Desa Digital
participants: @doohanas @adzharamrullah @yoslie @maulanayuseph @Ibwedagama

### CODE REVIEW

File: components/Registration/Mitra/index.vue [hunk 1]
```diff
@@ -393,6 +416,47 @@ export default {
       } else {
         this.isDropdownOpen = false
       }
+    },
+    fetchUser (token) {
+      try {
+        this.$axios.get('https://www.googleapis.com/oauth2/v2/userinfo', {
```
Comment (yoslie) [inline components/Registration/Mitra/index.vue:422]: discuss:
should this url saved on env or not? wdyt

**Review by adzharamrullah — APPROVED** [review]

File: components/Registration/Mitra/index.vue (hunk 1 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue:422]: I agree to save this URL in env kak @yoslie

File: components/Registration/Mitra/index.vue [hunk 2]
```diff
@@ -333,6 +341,21 @@ export default {
...
+    const urlParams = window.location.hash
+    if (urlParams) {
+      const queryparams = urlParams.split('#')[1]
+      const params = queryparams.split('&')
+      let pair = null
+      const data = []
+      params.forEach(function (d) {
+        pair = d.split('=')
+        data.push({ key: pair[0], value: pair[1] })
+      })
```
Comment (Ibwedagama) [inline components/Registration/Mitra/index.vue:354]: hello @doohanas , can't we achieve the same result using `this.$route` ? I believe it returns an object that contains the information of the URL so we don't have to split the URL strings 🤔

File: components/Registration/Mitra/index.vue [hunk 3]
```diff
@@ -393,6 +416,47 @@ export default {
...
+            Accept: 'application/json',
+            'Content-Type': 'application/json;charset=UTF-8'
+          }
+        })
+          .then(({ data }) => {
+            this.form.name = data.name
+            this.form.email = data.email
+          })
+      } catch (err) {
+        if (err.responsense?.status === 401) {
```
Comment (maulanayuseph) [inline components/Registration/Mitra/index.vue:434]: I think this typo should be `err.response` 🤔 ?

**Review by maulanayuseph — APPROVED** [review]

**Review by Ibwedagama — APPROVED** [review]

File: components/Registration/Mitra/index.vue (hunk 3 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue:434]: 😆 I think i was typo kak

Comment (doohanas) [reply components/Registration/Mitra/index.vue:434]: 😆 I think i was typo kak

File: components/Registration/Mitra/index.vue (hunk 2 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue:354]: Hello kak @Ibwedagama, yes, you're right I can achieve that with `this.$route` but I have the same issue after the user login and redirect to another page and the state is (see picture below)
image.
And I have the same issue with [this](https://github.com/nuxt-community/auth-module/discussions/1502). 
So, the solution, for now, I split the URL strings and get the `access_token` and pass in `fetchUser` method

File: components/Registration/Mitra/index.vue (hunk 3 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue:434]: 😆 I think i was typo kak

Comment (doohanas) [reply components/Registration/Mitra/index.vue:434]: 😆 I think i was typo kak

File: components/Registration/Mitra/index.vue (hunk 2 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue:354]: Hello kak @Ibwedagama, yes, you're right I can achieve that with `this.$route` but I have the same issue after the user login and redirect to another page and the state is (see picture below)
image.
And I have the same issue with [this](https://github.com/nuxt-community/auth-module/discussions/1502). 
So, the solution, for now, I split the URL strings and get the `access_token` and pass in `fetchUser` method

File: components/Registration/Mitra/index.vue (hunk 3 above)
Comment (doohanas) [reply components/Registration/Mitra/index.vue]: You can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/79/commits/a7e7a8c94fee3e482cb5c3256bd8aed3cd2aaa29)

File: components/Registration/Mitra/index.vue (hunk 2 above)
Comment (Ibwedagama) [reply components/Registration/Mitra/index.vue:354]: I see, okay just stick with your solution bro 👍🏼 👍🏼

**Review by yoslie — APPROVED** [review]

**Pull request merged**